### ISSUE TITLE
Increase code coverage to at least 80%

### ISSUE THREAD

**Issue by Ellzord** [Ellzord/JALSE#82]
Currently at 8.82% https://coveralls.io/builds/2526982 #25

**Comment by pbremer** [comment]
It should also be noted that if anyone makes any new claases/methods that they should also be effectively unit tested. If anyone does not know how to use Junit or what an effective unit test is just message me

**Comment by Ellzord** [comment]
Yup as much as possible. As far as test quality goes I'll settle for low right now as we have such little coverage. When bugs come in I think it should be part of the process to ensure there was a test that would of picked it up and that passes too (little ambitious but useful).

**Comment by Ellzord** [comment]
@wlmiller 
COVERAGE INCREASED (+36.02%) TO 44.84%
https://coveralls.io/builds/2555690

Nice one!

**Comment by wlmiller** [comment]
:+1:

**Comment by Ellzord** [comment]
@wlmiller 
COVERAGE INCREASED (+4.17%) TO 49.01%
https://coveralls.io/builds/2566611

~50% coverage is not too bad at all!

**Comment by Ellzord** [comment]
@wlmiller 
COVERAGE INCREASED (+11.15%) TO 60.16%
https://coveralls.io/builds/2580017

**Comment by Ellzord** [comment]
COVERAGE DECREASED (-0.11%) TO 70.06%
https://coveralls.io/builds/2639372

**Comment by Ellzord** [comment]
@DTing
COVERAGE INCREASED (+0.69%) TO 70.15%
https://coveralls.io/builds/3014767
